### PR TITLE
Attributes helpers

### PR DESCRIPTION
**Pull request by Abyss777**
Added helpers to `Extensible` class

Used them where needed.

Actually, `getLong` is never used, added just to consistence.

### CODE REVIEW

File: src/org/traccar/reports/ReportUtils.java [hunk 1]
```diff
@@ -63,18 +63,17 @@ public static double calculateDistance(Position firstPosition, Position lastPosi
...
         double lastOdometer = 0.0;
-        if (firstPosition.getAttributes().containsKey(Position.KEY_ODOMETER)) {
-            firstOdometer = ((Number) firstPosition.getAttributes().get(Position.KEY_ODOMETER)).doubleValue();
-        }
-        if (lastPosition.getAttributes().containsKey(Position.KEY_ODOMETER)) {
-            lastOdometer = ((Number) lastPosition.getAttributes().get(Position.KEY_ODOMETER)).doubleValue();
-        }
+        firstOdometer = firstPosition.getDouble(Position.KEY_ODOMETER);
+
+
```
Comment (tananaev) [inline src/org/traccar/reports/ReportUtils.java]: Too many spaces here.

**Comment by tananaev** [comment]
Looks good. Can you please check if it affects any of the database or JSON conversion of the model objects. Maybe you've already checked it?

**Comment by Abyss777** [comment]
Removed lines.

I checked it in:
- Project tests
- `test-generator.py`, it generates all events correctly and send email.
- Generated all reports of all types in web
- Check objects in DB visually

There was some changes in protocol encoders, if thew was covered by tests it is passed.

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**